docs/source/structure.rst

```diff
@@ -6,8 +6,8 @@ CAMELS data is structured as follows. CAMELS contains 2 simulations suites: Illu
 
 The scheme below shows the general structure of CAMELS data:
 
-.. image:: Sims_scheme.pdf
-
+ .. image:: Sims_scheme.png
```

Comment:
I suggest using this PNG or a similar one with a higher resolution so that it can be soon the the docs page.